map/src/styling/index.ts

```diff
@@ -14,8 +14,23 @@ const {
 export { css, createGlobalStyle, keyframes, ThemeProvider, THEME };
 export default styled;
 
+const LARGE_BREAKPOINT = 1000;
```

Comment:
We have a file here with all the breakpoints, maybe we can derive a way to use this instead? - https://github.com/reach4help/reach4help/blob/master/web-client/src/constants/mediaQueries.ts

Reply:
Ah, sharing between `web-client` and `map` requires introducing a new package with a compile step (or using symlinks). I think it's something we want to do, but probably shoudln't block this going in based on sharing of these constants (in particular given that the breakpoints are different right now).

Reply:
Yeah, I agree. I'll open up a new issue though so we are aware of this later